src/main/scala/nl.knaw.dans.easy.solr4files/components/AuthInfoItem.scala

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.nio.file.{ Path, Paths }
+import java.text.SimpleDateFormat
+import java.util.UUID
+
+import nl.knaw.dans.easy.solr4files._
+import org.joda.time.DateTime
+import org.json4s.ext.{ EnumNameSerializer, JodaTimeSerializers }
+import org.json4s.native.JsonMethods.parse
+import org.json4s.{ DefaultFormats, Formats }
+
+import scala.util.{ Failure, Try }
+
+/** The class arguments must match the json fields returned by the service easy-auth-info */
+case class AuthInfoItem(itemId: String,
```

Comment:
Every client of our AuthInfo mode might want to use this class. 
But the more general question is if we are making client libs for our modules, guess not?

Reply:
if you mean...
1. a library for common abstractions in our microservices - _I'm not sure if you would add this in there_
2. a library for a client to communicate with the service (think the Fedora API) - _I'm definitely in for that, but don't really see the added value of this for now, especially given our team is fairly small and our API's are relatively simple and not widely used in the organisation (although easy-bag-store is approaching that state...). If you're the size of NetFlix this would make more sense._
3. something else - _I'm sorry, I didn't understand what you meant._